### PR TITLE
chore(web): add useManageSwitchState

### PR DESCRIPTION
**Pull request by atnagata**
# Overview
add useManageSwitchState
## How I tested
Confirmation of activation assuming multiple component (button) state use

### CODE REVIEW

File: web/src/beta/hooks/useManageSwitchState/hooks.test.tsx [hunk 1]
```diff
@@ -0,0 +1,57 @@
+import { renderHook, act } from "@testing-library/react";
+import { expect, test } from "vitest";
+
+import useManageSwitchState, { Props } from "./hooks";
+
+type SwitchField = {
+  id: string;
+  active: boolean;
```
Comment (keiya01) [inline web/src/beta/hooks/useManageSwitchState/hooks.test.tsx:8]:
```suggestion
  name: string;
```
I think the props has `id` and `active` field in default, so could you set `name` to this field?
Also could you check if the name field is included in the fields state which is returned from the hooks?

Comment (atnagata) [reply web/src/beta/hooks/useManageSwitchState/hooks.test.tsx]: Added and committed.

File: web/src/beta/hooks/useManageSwitchState/hooks.tsx [hunk 2]
```diff
@@ -2,6 +2,7 @@ import { useState, useCallback } from "react";
 
 type SwitchField<T> = {
   id: string;
+  name?: string;
```
Comment (keiya01) [inline web/src/beta/hooks/useManageSwitchState/hooks.tsx:5]: これはいらないです

**Review by KaWaite — APPROVED** [review]
After @keiya01 's comment is addressed and resolved you can merge!

**Pull request merged**